CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## \[Unreleased\]
 
+### Changed
+
+- test: pass the CI e2e tests on Python 3.10
```

Comment:
Please add the PR number in the changelog (#56)

Reply:
Done, thanks :)